README.md

```diff
@@ -61,10 +61,8 @@ This is a list of shortcuts to some interesting topics:
 
 * :rocket: If you want to learn how to **contribute new recipes**, please read [docs/adding_packages/](docs/adding_packages/README.md).
 * :speech_balloon: **FAQ**: most common questions are listed in [docs/faqs.md](docs/faqs.md).
-* :warning: The conan-center **hook errors** reported by CCI Bot can be found in the [docs/error_knowledge_base.md](docs/error_knowledge_base.md).
```

Comment:
note that this is a pending task - that the hooks report in the checks tab links to a detailed explanation where more context is necessary - but good to remove this for now